hc_core/src/common/entry.rs

```diff
@@ -32,6 +32,11 @@ impl Entry {
     pub fn content(&self) -> String {
         self.content.clone()
     }
+
+    pub fn validate(&self) -> bool {
+        // always valid iff immutable and new() enforces validity
```

Comment:
iffffff

Reply:
if and only if

Reply:
😅 Sorry. Lesson learned!

Reply:
it certainly is an odd word that looks like a mistake 😬